javascript-review.js

```diff
@@ -168,9 +300,36 @@
 // Using the following API endpoint access the Australian flag svg link and console.log it
 // => https://restcountries.eu/rest/v2/all
 
+function onResponse(resp) {
+    resp.json().then(data => console.log(data))
```

Comment:
this is how it can be done 

```js
fetch('https://restcountries.eu/rest/v2/all')
  .then((response) => response.json())
  .then((data) => {
    const australia = data.find((country) => {
      return country.name == "Australia"
    })
    console.log(australia.flag)
})
```